cmd2/plugin.py

```diff
@@ -0,0 +1,27 @@
+#
+# coding=utf-8
+"""Classes for the cmd2 plugin system"""
+import attr
```

Comment:
At some point did Python incorporate the 3rd-party ``attrs`` module into the standard library as built-in module ``attr``??

If so, are we sure it is properly available in Python 3.4 without any additional dependencies?

Reply:
No, but I forgot to add it to `setup.py`. Fixed. Primary interest in this library is to make (almost) immutable `Statement` object.